build_defs/python.build_defs

```diff
@@ -558,7 +558,7 @@ def python_wheel(name:str, version:str, hashes:list=None, package_name:str=None,
 
         cmd += ['if [ "$FOUND" = true ]']
         cmd += ['then wget -O "$OUTS" "$DOWNLOAD"']
-        cmd += ['else $TOOL --package ' + package_name + ' --version ' + version + '; fi']
+        cmd += ['else $TOOL --package ' + package_name + ' --version ' + version + ' --index ' + CONFIG.PYTHON.WHEEL_REPO + '; fi']
```

Comment:
the wheel repo isn't exactly the same as a PyPI repo; we just look for wheels in it by filename, whereas IIRC PyPI has an HTML-based index.

Don't think this change is necessarily bad, but I don't think it will Just Work if you point a PyPI resolver against an existing wheel repo?